src/admin/index.js

```diff
@@ -285,6 +307,30 @@ module.exports = ({
     return setOffsets({ groupId, topic, partitions: partitionsToSeek })
   }
 
+  /**
+   * @param {string} groupId
+   * @param {string} topic
+   * @param {number} timestamp
+   * @return {Promise}
+   */
+  const resetOffsetsByTimestamp = async ({ groupId, topic, timestamp }) => {
```

Comment:
How do you feel about putting `timestamp` as an optional parameter to `resetOffsets` instead of introducing a new public method (it could internally still delegate to this method, I'm not saying the actual implementation has to live there)? I'm not sure myself which is the lesser evil - introducing more public methods or having several optional parameters.

Reply:
I agree it seems not a good idea to introduce a new public method. Since the body of `resetOffsetsByTimestamp` being quite simple, I'll just put it into the document under `Admin.md` and remove the function.